contrib/!lang/erlang-elixir/README.md

```diff
@@ -87,7 +102,16 @@ Note that `EDTS` is disable on Windows.
 <kbd>SPC m m i</kbd>  | Start `iex` in the context of an Elixir project
 <kbd>SPC m m x</kbd>  | Run the given expression in the Elixir application context
 
-#### Elixir Evaluation in place
+##### Project
+
+    Key Binding       |                 Description
+----------------------|------------------------------------------------------------
+<kbd>SPC m p f</kbd>  | Open project test directory and list all test files.
+<kbd>SPC m p c</kbd>  | Open the appropriate test file for the current buffer file in a new window.
+<kbd>SPC m p t</kbd>  | Toggle between a file and its tests in the current window.
+<kbd>SPC m p T</kbd>  | Toggle between a file and its tests in other window.
+
```

Comment:
This goto should go under `SPC m g`.
`SPC m g t` to go to test
I would use the `SPC m g a` convention (alternate file) to switch back and forth between the tests and the implementation

So my suggestions:
`SPC m p f` ---> `SPC m p t`

`SPC m p t` ---> `SPC m g t`
`SPC m p T` ---> `SPC m g T`

I'm not sure about the usefulness of `SPC m p c`, this should be a fall back when toggling (the first time it is used when the test file is not yet opened).